concourse/steps/release.py

```diff
@@ -883,6 +886,7 @@ def release_and_prepare_next_dev_cycle(
     release_commit_message_prefix: str=None,
     slack_channel_configs: list=[],
     version_operation: str='bump_minor',
+    release_notes_creation: str='default',
```

Comment:
musn't this match enum field-name (thus uppercase)?

Reply:
No, as with vanilla-Enums these `EnumWithDocumentation` can also be instantiated with the value.

Reply:
just check again, I mixed up `EnumValueWithDocumentation` and `EnumWithDocumentation`
you're right, lowercase 'default' is fine